src/Microsoft.Azure.WebJobs.Host/DefaultNameResolver.cs

```diff
@@ -11,14 +12,21 @@ namespace Microsoft.Azure.WebJobs
     /// </summary>
     public class DefaultNameResolver : INameResolver
     {
+        private readonly IConfiguration _configuration;
+
+        public DefaultNameResolver(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
         /// <summary>
         /// Resolves tokens by looking first in App Settings and then in environment variables.
         /// </summary>
         /// <param name="name">The token to resolve.</param>
         /// <returns>The token value from App Settings or environment variables. If the token is not found, null is returned.</returns>
         public virtual string Resolve(string name)
         {
-            return ConfigurationUtility.GetSetting(name);
+            return _configuration[name];
         }
```

Comment:
The DefaultNameResolver now goes directly against IConfiguration. ConfigurationUtility should be removed, but for here I just will avoid using it.

Reply:
Thanks! We should probably mark that type as obsolete with a proper description. We want to start moving away from it as it really provides no value today.